warehouse/ingest-core/src/main/java/datawave/ingest/metadata/EventMetadata.java

```diff
@@ -216,6 +216,11 @@ protected void addEventField(IngestHelperInterface helper, RawRecordContainer ev
             return;
         }
         
+        if (helper.isShardExcluded(fieldName)) {
+            log.debug("{} is an excluded field, not adding to event", fieldName);
```

Comment:
Should this message be "not adding to event **fields**"?

Reply:
no, it should say "not adding event or frequency metadata columns," but I was just following precedent on lines 215 and 210.